ldoc/html/ldoc_ltp.lua

```diff
@@ -182,8 +182,8 @@ return [==[
 #      for value in iter(tag) do
          $(li)$(custom.format and custom.format(value) or M(value))$(il)
 #      end -- for
+    </ul "@todo">
```

Comment:
What is this?

Reply:
Not sure where the @todo comes from. I should have read through the file once more.